src/actions/receive-profile.js

```diff
@@ -1407,7 +1407,10 @@ export function retrieveProfileFromFile(
     dispatch(waitingForProfileFromFile());
 
     try {
-      if (file.type === 'application/zip') {
+      if (
+        file.type === 'application/zip' ||
+        file.type === 'application/x-zip-compressed'
```

Comment:
Have you got problems with this? I'm asking because it's not especially the goal for this patch, even though I'm not opposed.

Reply:
Yeah, I wanted to test '*' in the zip file viewer for regressions, but I wasn't able to upload any zip files whatsoever. It would error out saying the buffer wasn't utf-8. Upon a cursory glance, it's because the files uploaded by me had this deprecated MIME type instead of 'application/zip'. I meant to post a separate PR for this but accidentally committed it to this branch, will revert this change and post it separately later.